spec/models/time_based_repo_rater_spec.rb

```diff
@@ -9,125 +9,125 @@
-        repo = Repository.new('joe-bloggs', 'repo-name')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'repo-name')
         repo.add_commit(commit)
         repo
       end
 
       it 'should rate maximum value' do
-        expect(subject.rate(repo)).to eq(10)
+        expect(subject.rate).to eq(10)
       end
     end
 
     context 'repo with commit within 1 week' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', time_days_ago(7), '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', time_days_ago(7), '') }
       let(:repo) do
-        repo = Repository.new('joe-bloggs', 'repo-name')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'repo-name')
         repo.add_commit(commit)
         repo
       end
 
       it 'should rate maximum value' do
-        expect(subject.rate(repo)).to eq(10)
+        expect(subject.rate).to eq(10)
       end
     end
 
     context 'repo with commit within 1 month' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', time_days_ago(30), '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', time_days_ago(30), '') }
       let(:repo) do
-        repo = Repository.new('joe-bloggs', 'repo-name')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'repo-name')
         repo.add_commit(commit)
         repo
       end
 
       it 'should rate something high' do
-        expect(subject.rate(repo)).to eq(8)
+        expect(subject.rate).to eq(8)
       end
     end
 
     context 'repo with commit within 3 months' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', time_months_ago(3), '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', time_months_ago(3), '') }
       let(:repo) do
-        repo = Repository.new('joe-bloggs', 'repo-name')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'repo-name')
         repo.add_commit(commit)
         repo
       end
 
       it 'should rate high-med' do
-        expect(subject.rate(repo)).to eq(6)
+        expect(subject.rate).to eq(6)
       end
     end
 
     context 'repo with commit within 6 months' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', time_months_ago(6), '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', time_months_ago(6), '') }
       let(:repo) do
-        repo = Repository.new('joe-bloggs', 'repo-name')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'repo-name')
         repo.add_commit(commit)
         repo
       end
 
       it 'should rate low-med' do
-        expect(subject.rate(repo)).to eq(4)
+        expect(subject.rate).to eq(4)
       end
     end
 
     context 'repo with commit within 12 months' do
-      let(:commit) { Commit.new('foo', 'joe bloggs', time_months_ago(12), '') }
+      let(:commit) { Commit.build('foo', 'joe bloggs', time_months_ago(12), '') }
       let(:repo) do
-        repo = Repository.new('joe-bloggs', 'repo-name')
+        repo = Repository.from_owner_and_name('joe-bloggs', 'repo-name')
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.